components/raftstore/src/store/fsm/peer.rs

```diff
@@ -11,7 +11,7 @@ use std::{
     },
     iter::{FromIterator, Iterator},
     mem,
-    sync::{Arc, Mutex},
+    sync::{Arc, Mutex, mpsc as other_mpsc},
```

Comment:
What about just importing `sync::{Arc, Mutex, mpsc::SyncSender}` rather than renaming it?

Reply:
> What about just importing `sync::{Arc, Mutex, mpsc::SyncSender}` rather than renaming it?

get it